org.eclipse.xtext.doc/contents/350_continuous_integration.html

```diff
@@ -313,6 +313,14 @@ <h2 id="maven-tycho-hints">Maven Tycho Hints</h2>
     </tr>
   </thead>
   <tbody>
+    <tr>
+      <td><a href="https://download.eclipse.org/modeling/tmf/xtext/updates/releases/2.35.0/">2.35.0</a></td>
+      <td><a href="https://download.eclipse.org/modeling/emf/emf/builds/release/2.38.0">2.38.0</a> (2.29.0)</td>
```

Comment:
Are the numbers un parenthesis in this part correct? They don't correspond to the numbers in the update sites.

Reply:
there are the minimal versions

Reply:
see also original here

https://github.com/eclipse/xtext-website/blob/main/xtext-website/documentation/350_continuous_integration.md